test/integration/logeater_test.rb

```diff
@@ -60,7 +60,24 @@ class LogeaterTest < ActiveSupport::TestCase
 
     should "create an entry in the database" do
       assert_difference "Logeater::Request.count", +1 do
-        reader.import
+        logfile_reader.import
+      end
+    end
+  end
+
+
+  context "Given an app and a timestamp, import_since" do
+    setup do
+      log_sample = File.open(File.expand_path("./test/data/single_request.log"))
+      log_sample.lines do |line|
+        Logeater::Event.create(ep_app: app, original: line)
+      end
+      @events = Logeater::Event.all
+    end
+
+    should "import events since that given timestamp buuga" do
```

Comment:
👆 What's **buuga** in this test name?

Reply:
😳 
That's my code word when I want to run a single test.. Now everyone knows... :man_facepalming: